src/clib/pio.h

```diff
@@ -27,6 +27,11 @@
     (((NC_VERSION_MAJOR == Maj) && (NC_VERSION_MINOR == Min) && (NC_VERSION_PATCH >= Pat)) || \
      ((NC_VERSION_MAJOR == Maj) && (NC_VERSION_MINOR > Min)) || (NC_VERSION_MAJOR > Maj))
 
+#if defined(NC_FillValue)
+#define NCDF_FillValue NC_FillValue
+#elif defined(_FillValue)
+#define NCDF_Fillvalue _FillValue
```

Comment:
I think instead of introducing a third macro, just define NC_FillValue if not defined and use that everywhere.

Reply:
@WardF I've made the change suggested by @edwardhartnett can you give me permission to push back to your fork?

Reply:
@jedwards4b sorry for the radio silence, I've been at an NSF Unidata retreat for the last couple of days.  I'll modify that permission!

Reply:
@jedwards4b The 'allow edits by maintainers' option is selected, does this not give you access to push changes to this PR? If not, maybe it would be easier (and certainly no problem from my end) to pull the changes I've proposed into a new PR that you can push and merge?